sound/soc/sof/ipc4-topology.c

```diff
@@ -427,6 +427,62 @@ static void sof_ipc4_widget_free_comp_pcm(struct snd_sof_widget *swidget)
 	swidget->private = NULL;
 }
 
+static int sof_ipc4_widget_setup_comp_buffer(struct snd_sof_widget *swidget)
```

Comment:
I'd modify the commit description to say "Module copier is a buffer-like component with demuxing capabilities"